src/LibraryViewExtensionWebView2/Properties/AssemblyInfo.cs

```diff
@@ -0,0 +1,9 @@
+﻿using System.Reflection;
+using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
+
+
+[assembly: AssemblyTitle("LibraryViewExtensionWebView2")]
+// The following GUID is for the ID of the typelib if this project is exposed to COM
```

Comment:
You may also want to add the same `[assembly: InternalsVisibleTo("ViewExtensionLibraryTests")]` in order to add tests later for this view extension